assets/cms/js/tree.js

```diff
@@ -218,7 +218,25 @@ ConcreteTree.prototype = {
 
                 return true
             },
-
+            expand: function(event, data) {
+                //only if 'selected' array is available execute code.
+                if (options.ajaxData.selected) {
+                    //since the expand options is triggered we can asume that there are children.
+                    data.node.children.forEach(function(nodeChild){
+                        if (options.ajaxData.selected.includes(nodeChild.key)) {
+                            nodeChild.setSelected(!0);
```

Comment:
What about using `true` instead of `!0` ?

Reply:
That is better indeed. I tested with the minified one from the core so I copied it from there. I will change this to `true`.